packages/Checkbox/src/checkbox.scss

```diff
@@ -1,291 +1,367 @@
+    --ids-checkbox-size: 1rem;
+    --ids-checkbox-radius: var(--hop-shape-rounded-sm);
+    --ids-checkbox-check: var(--hop-neutral-icon-active);
+    --ids-checkbox-margin: var(--hop-space-inline-sm);
+    --ids-checkbox-top-space: 0;
+    --ids-checkbox-box-sizing: border-box;
+    --ids-checkbox-text: var(--hop-neutral-text);
+    --ids-checkbox-text-disabled: var(--hop-neutral-text-disabled);
+    --ids-checkbox-background: var(--hop-neutral-surface);
+    --ids-checkbox-background-hover: var(--hop-neutral-surface-hover);
+    --ids-checkbox-background-disabled: var(--hop-neutral-surface-disabled);
+    --ids-checkbox-background-checked: var(--hop-neutral-surface-active);
+    --ids-checkbox-background-checked-disabled: var(--hop-neutral-surface-disabled);
+    --ids-checkbox-border: var(--hop-neutral-border);
+    --ids-checkbox-border-hover: var(--hop-neutral-border-hover);
+    --ids-checkbox-border-checked: var(--hop-neutral-border-active);
+    --ids-checkbox-border-checked-hover: var(--hop-neutral-border-active);
+    --ids-checkbox-border-disabled: var(--hop-neutral-border-disabled);
+    --ids-checkbox-check-width: .25rem;
+    --ids-checkbox-check-height: .5rem;
+    --ids-checkbox-check-border: .125rem;
+    --ids-checkbox-check-left: .0625rem;
+    --ids-checkbox-check-top: .4375rem;
+    --ids-checkbox-check-color: var(--hop-neutral-icon-active);
+    --ids-checkbox-check-color-disabled: var(--hop-neutral-icon-disabled);
+    --ids-checkbox-indeterminate-width: .5625rem;
+    --ids-checkbox-indeterminate-border: .09375rem;
+
+    /* Completion */
+    --ids-checkbox-completion-size: 1.5rem;
+    --ids-checkbox-completion-background-disabled: var(--hop-neutral-surface-disabled);
+    --ids-checkbox-completion-border-disabled: var(--hop-neutral-border-disabled);
+    --ids-checkbox-completion-check-size: .75;
+    --ids-checkbox-completion-check-color-hover: transparent;
+    --ids-checkbox-completion-check-color-selected: var(--hop-neutral-icon-active);
+    --ids-checkbox-completion-check-color-default: transparent;
+    --ids-checkbox-completion-check-color-disabled: var(--hop-neutral-icon-disabled);
 }
 
 %check {
-  width: var(--ids-checkbox-check-width);
-  height: var(--ids-checkbox-check-height);
-  border: .2rem solid currentcolor;
-  border-bottom: 0;
-  border-left: 0;
-  transform: scaleX(-0.9) rotate(135deg);
-  transform-origin: top left;
+    width: var(--ids-checkbox-check-width);
+    height: var(--ids-checkbox-check-height);
+    border: var(--ids-checkbox-check-border) solid currentcolor;
+    border-bottom: 0;
+    border-left: 0;
+    color: var(--ids-checkbox-check-color, currentcolor);
+    transform: scaleX(-0.9) rotate(135deg);
+    transform-origin: top left;
 }
 
 %indeterminate {
-  width: .6rem;
-  border: .2rem solid currentcolor;
-  border-top: 0;
-  border-left: 0;
+    width: var(--ids-checkbox-indeterminate-width, .6rem);
```

Comment:
this won't compute the same between both themes as the rem is not on the same base.